custom_components/dyson_local/sensor.py

```diff
@@ -256,6 +199,8 @@ class DysonTemperatureSensor(DysonSensorEnvironmental):
     """Dyson temperature sensor."""
 
     _SENSOR_TYPE = "temperature"
+    _SENSOR_NAME = "Temperature"
+    _attr_device_class = DEVICE_CLASS_TEMPERATURE
 
```

Comment:
The particulates sensor should probably be added here as specified here: https://github.com/shenxn/libdyson/blob/main/libdyson/dyson_pure_hot_cool_link.py and there: https://github.com/shenxn/libdyson/blob/70d890259fe7baaa106f8c3302cd42077697ddd7/libdyson/dyson_pure_cool_link.py#L7

Reply:
Ah it looks like the Link devices don't have pm25/pm10 distinctions. Sorry about that, let me fix it.

Reply:
Okay this should do it, although I'm not quite sure on whether this is PM1 or PM10 that it reports.

Also, added the focus mode and tilt entities, because that was easy once I was looking at it.

Reply:
It reports both pm2.5 and pm10 so I guess the best way would be to pass it as pm10 as *technically*, pm10 also encompasses pm2.5

Reply:
I think right now @shenxn's component reports the same value for both… but libdyson doesn't specify which level it is, and even says "unknown unit."

Do you happen to have a screenshot from the app to compare how it's reported there, if at all? Otherwise for now I guess reporting it as PM1 will at least do something… but admittedly to merge upstream it might be better to know exactly what's going on there.

Reply:
The Air Quality on my Dyson Pure Hot+Cool Link has no numeric value in the app and no associated unit. It presents as a simple ordinal scale.

Reply:
I can confirm, on my app I have the same issue, only a general "air quality" graph no units to be found